Dockerfile

```diff
@@ -6,9 +6,17 @@ LABEL org.opencontainers.image.vendor="Adfinis"
 COPY backup.sh /usr/local/bin/backup.sh
 
 RUN microdnf update -y && rm -rf /var/cache/yum
-RUN microdnf install findutils curl tar gzip -y && microdnf clean all
-RUN curl -L https://github.com/etcd-io/etcd/releases/download/v3.5.11/etcd-v3.5.11-linux-amd64.tar.gz \
+RUN microdnf install -y \
+	curl-7.61.1-34.el8 \
```

Comment:
I'd honestly say specifying the exact versions is not worth the effort. As long as we're regularly updating the source image and we're just installing basic CLI tools I don't see the benefit of it.